description: gcc does not allow negative values for size_t types
author: Michael Gilbert <mgilbert@debian.org>

--- a/base/containers/intrusive_heap.h
+++ b/base/containers/intrusive_heap.h
@@ -149,7 +149,7 @@ namespace base {
 // in place.
 class BASE_EXPORT HeapHandle {
  public:
-  enum : size_t { kInvalidIndex = -1 };
+  enum : size_t { kInvalidIndex = std::numeric_limits<size_t>::max() };
 
   constexpr HeapHandle() = default;
   constexpr HeapHandle(const HeapHandle& other) = default;
